src/modules/micrortps_bridge/CMakeLists.txt

```diff
@@ -40,20 +40,27 @@ if(NOT FASTRTPSGEN)
 	message(FATAL_ERROR "Unable to find fastrtpsgen")
 endif()
 
-set(config_rtps_send_topics
-	sensor_combined
-	sensor_baro
-	sensor_selection
-	vehicle_gps_position
-	airspeed
-	vehicle_land_detected
-	vehicle_status
-   )
-
-set(config_rtps_receive_topics
-   sensor_baro
-   optical_flow
-   )
+if (EXISTS "${PX4_SOURCE_DIR}/msg/tools/uorb_rtps_message_ids.yaml")
+
+  set(config_rtps_send_topics)
+	execute_process(
+		COMMAND ${PYTHON_EXECUTABLE} ${PX4_SOURCE_DIR}/msg/tools/uorb_rtps_classifier.py -s
+		OUTPUT_VARIABLE config_rtps_send_topics
```

Comment:
Can we integrate this into the build step? Having it like this will break incremental builds for updated topics/id's.

Reply:
I am not sure of what you are suggesting here. Can you be more specific on what you are asking for? Thanks.

Reply:
This runs at the configure step and not at build step. But it should be run at build time, which means you cannot use the `config_rtps_send_topics` cmake variable.
From what I see this list is given to `generate_microRTPS_bridge.py` as argument (transformed to file names), so instead we can directly pass the yaml path to `generate_microRTPS_bridge.py` which then extracts the list. But I have not looked at that in detail, you might have a better solution.
I would like to do only as little as really required in cmake (it quickly becomes hard to trace & debug).

Reply:
@bkueng I understand what you mean: the problem is that the client side also requires this list, in specific, the uORB CDR serialization sources and headers. Unless `px_generate_uorb_topic_files.py` also creates a `Classifier`object as well.

Reply:
@bkueng I addressed your other requests. I would like to leave this one to the next iteration. Can you please review again and merge if all ok? Thanks

Reply:
> Unless px_generate_uorb_topic_files.py also creates a Classifierobject as well.

Let's do that then. Next iteration is fine.

Reply:
> From what I see this list is given to `generate_microRTPS_bridge.py` as argument (transformed to file names), so instead we can directly pass the yaml path to `generate_microRTPS_bridge.py` which then extracts the list. 

@bkueng Actually no, the list of file names is used by `px_generate_uorb_topic_files.py`, which is not dependent of the agent side, it's all client related. The problem here is mainly related to how we are going to give the list of files to build to CMake. We can have a file list to be generated on the Python script already, but we still need to give a list of files that compose a target on CMake. And the only way I am seeing it right now is to execute a command that gives me those and make them available as a CMake variable.